go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/containerd/nerdctl/v2
 
-go 1.21
+go 1.21.0
```

Comment:
I prefer enforcing the last patch version for 1.21 as minimum supported version

Reply:
Sure. Will do that with next push.